src/main/scala/mesosphere/marathon/api/MarathonRestModule.scala

```diff
@@ -56,26 +47,10 @@ class MarathonRestModule extends ServletModule {
     bind(classOf[v2.SchemaResource]).in(Scopes.SINGLETON)
```

Comment:
Just so I understand. The next step is to use our module based DI instead of Guice here as well, right?

Reply:
Yes; right now we remove jersey-guice. This is a pretty big step towards getting rid of guice generally.